libqtile/backend/wayland/core.py

```diff
@@ -748,17 +749,21 @@ def get_screen_info(self) -> List[Tuple[int, int, int, int]]:
 
     def grab_key(self, key: Union[config.Key, config.KeyChord]) -> Tuple[int, int]:
         """Configure the backend to grab the key event"""
-        keysym = xkb.keysym_from_name(key.key, case_insensitive=True)
+        #keysym = xkb.keysym_from_name(key.key, case_insensitive=True)
+        keysyms = self.multilang_mapping.get_keysyms(key)
         mask_key = wlrq.translate_masks(key.modifiers)
-        self.grabbed_keys.append((keysym, mask_key))
-        return keysym, mask_key
+        for keysym in keysyms:
+            self.grabbed_keys.append((keysym, mask_key))
+        return keysyms, mask_key
 
     def ungrab_key(self, key: Union[config.Key, config.KeyChord]) -> Tuple[int, int]:
```

Comment:
Please update the return type annotation here to reflect the multiple keysyms that could be returned. Same above with `grab_key`